server/game/core/ability/PlayerOrCardAbility.js

```diff
@@ -107,9 +107,20 @@ class PlayerOrCardAbility {
             case TargetMode.Select:
                 return new SelectTargetResolver(name, properties, this);
             case TargetMode.Player:
+            case TargetMode.MultiplePlayers:
                 return new PlayerTargetResolver(name, properties, this);
-            default:
+            case TargetMode.AutoSingle:
+            case TargetMode.Exactly:
+            case TargetMode.ExactlyVariable:
+            case TargetMode.MaxStat:
+            case TargetMode.Single:
+            case TargetMode.Unlimited:
+            case TargetMode.UpTo:
+            case TargetMode.UpToVariable:
+            case undefined: // CardTargetResolver contains behavior that defaults the mode to TargetMode.Single if it is not defined yet.
```

Comment:
Need to add an explicit case here for null as well because switch uses `===` comparison (as opposed to `==`) so null and undefined are different